### PR TITLE
Fix pb when number of impacted substations is too high on network modifications

### PR DESCRIPTION
**Pull request by FranckLecuyer**

### CODE REVIEW

**Review by laurgarn — APPROVED** [review]

File: src/components/network/map-equipments.js [hunk 1]
```diff
@@ -84,15 +85,21 @@ export default class MapEquipments {
         currentNode,
         substationsIds
     ) {
+        const substationsIdsToFetch =
+            substationsIds &&
+            substationsIds.length > MAX_NUMBER_OF_IMPACTED_SUBSTATIONS
```
Comment (flomillot) [inline src/components/network/map-equipments.js:90]:
```suggestion
            substationsIds?.length > MAX_NUMBER_OF_IMPACTED_SUBSTATIONS
```

File: src/components/network/network.js [hunk 2]
```diff
@@ -513,10 +514,15 @@ export default class Network {
         substationsIds
     ) {
         if (substationsIds) {
+            const substationsIdsToFetch =
+                substationsIds &&
+                substationsIds.length > MAX_NUMBER_OF_IMPACTED_SUBSTATIONS
```
Comment (flomillot) [inline src/components/network/network.js:519]: Same as above

**Review by flomillot — APPROVED** [review]

File: src/components/network/map-equipments.js (hunk 1 above)
Comment (FranckLecuyer) [reply src/components/network/map-equipments.js]: Done

File: src/components/network/network.js (hunk 2 above)
Comment (FranckLecuyer) [reply src/components/network/network.js]: Done

File: src/components/network/map-equipments.js [hunk 3]
```diff
@@ -84,15 +85,20 @@ export default class MapEquipments {
         currentNode,
         substationsIds
     ) {
+        const substationsIdsToFetch =
+            substationsIds?.length > MAX_NUMBER_OF_IMPACTED_SUBSTATIONS
+                ? undefined
+                : substationsIds; // TODO : temporary to fix fetching request failing when number of impacted substations is too high
+
```
Comment (SlimaneAmar) [inline src/components/network/map-equipments.js:92]: Maybe invert the test for the case if  `substationsIds` is undefined

Comment (SlimaneAmar) [reply src/components/network/map-equipments.js:92]: Right :(

**Pull request merged**